app/__init__.py

```diff
@@ -1,15 +1,14 @@
 # -*- coding: utf-8 -*-
 import os
-import sys
-import inspect
+from logging.handlers import RotatingFileHandler
+
 from flask import Flask
-from flask_user import UserManager, UserMixin
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
 from flask_wtf.csrf import CSRFProtect
 from flask_mail import Mail
 from flask_bootstrap import Bootstrap
-from config import DevelopmentConfig, ProductionConfig
+from config import DevelopmentConfig
```

Comment:
@joeyzhou98 ProductionConfig needs to be back in the import for the production VM to run